pages/templates/base.html

```diff
@@ -82,11 +82,39 @@
                     </span>
                 </div>
             </div>
-            <div class="row mb-5 mb-md-0 pb-4">
+            <div class="row mb-0 pb-4">
                 <div class="col-12">
                     <span class="text-secondary">&copy; 2020 Snowy Pier Games LLC. All rights reserved.</span>
                 </div>
             </div>
+            <div class="row mb-5 mb-md-0 pb-4">
+                <div class="col">
```

Comment:
See my comment on your .col class in feed.css about using "no-gutters" instead

Reply:
I don't see how that applies here, there are gutters if it's small

Reply:
actually there are always gutters in this row

Reply:
Disregard.  I saw that you had customized that .col class to have zero-padding, and assumed you didn't want padding here.  But you removed that class.